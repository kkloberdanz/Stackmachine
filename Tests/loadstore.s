PUSH
10
STORE
4
PUSH
6
LOAD
4
ADD
HALT
