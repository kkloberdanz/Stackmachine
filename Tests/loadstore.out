1
10
13
4
1
6
12
4
2
999999
